src/types/component-props/_mixins.ts

```diff
@@ -14,12 +14,35 @@ export type HeaderWithAnchorMixin = {
     toggleCopyButton: boolean;
 };
 
-export enum Audience {
+export enum Audiences {
     PERSON = 'person',
     EMPLOYER = 'employer',
     PROVIDER = 'provider',
 }
 
+export enum ProviderAudiences {
+    DOCTOR = 'doctor',
+    DENTIST = 'dentist',
+    OPTICIAN = 'optician',
+    OCCUPATIONAL_THERAPIST = 'occupational_therapist',
+    AID_SUPPLIER = 'aid_supplier',
+    MEASURES_ORGANIZER = 'measures_organizer',
+    ADMINISTRATOR = 'administrator',
+    MUNICIPALITY = 'municipality',
+    OTHER = 'other',
+}
+
+export type Audience = {
+    _selected: Audiences;
+    provider_audience?: ProviderAudiences;
+};
+export const getAudience = (audience: Audience) => {
+    if (audience && audience._selected) {
+        return audience._selected;
+    }
+    return Audiences.PERSON;
```

Comment:
Kanskje gjøre noe sånt midlertidig slik at den er bakoverkompatibel med forrige modell?
```suggestion
    return typeof audience === 'string' ? audience : Audiences.PERSON;
```